docgen/src/Styling.md

```diff
@@ -6,62 +6,174 @@ nav_groups:
 nav_sort: 1
 ---
 
+Welcome to the react-instantsearch theming tutorial!
+
+`react-instantsearch` aims to provide the best search experience your way! We believe that theming is a big part when comes the customization of the search experience. That's why react-instantsearch uses [`react-themeable`](https://github.com/markdalgleish/react-themeable) to provides easily customizable widgets. 
```

Comment:
We do not need to try to sale the library on every page, at some point we need to be a bit more down to earth, direct in our approach so that the user do not feel like "Okai, yeah, I know you are awesome"

Recommendation: drop the salesy speach, go directly to the core of what the user is searching for: styling use cases and solutions.